shiny/ui/_sidebar.py

```diff
@@ -177,6 +199,12 @@ class directly. Instead, supply the :func:`~shiny.ui.sidebar` object to
         * If four, then the values will be interpreted as top, right, bottom, and left
           respectively.
 
+    Methods
+    -------
+    open
+        Get or set the initial state of the sidebar. Returns a dataclass with `desktop`
+        and `mobile` attributes.
+
```

Comment:
I think that you can just put this info in the docstring for the `open` method instead of putting it here.